README.md

```diff
@@ -140,7 +165,7 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+Dear Warrior your RESTful journey lies between the http requests from applications.
```

Comment:
Clever.  :)  Can you elaborate a bit on what it means to be restful?  What do you expect from a RESTful API?  What routes?